### PR TITLE
Add XR Sepolia chain (#1)

### PR DESCRIPTION
**Pull request by RealWooblay**
## PR-Codex overview
This PR adds a new XR Sepolia chain to the project.

### Detailed summary
- Added `xrSepolia` chain definition to `chains/index.ts`
- Created `xrSepolia` chain definition in `chains/definitions/xrSepolia.ts` with ID 2730
- Configured `xrSepolia` with native currency, RPC URLs, block explorers, and set as a testnet

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: src/experimental/eip5792/actions/writeContracts.test.ts [hunk 1]
```diff
@@ -198,12 +198,12 @@ test('default', async () => {
               "0x000000000000000000000000f39fd6e51aad88f6f4ce6ab8827279cfffb92266",
               "0x0000000000000000000000000000000000000000000000000000000000000221",
             ],
-            "transactionHash": "0xf0c48f098fb7f44a4ca942f85277e97ad3481020e21f1f165eed514205a21694",
+            "transactionHash": "0x217f9a45ed86f0f83ec4215b2bb7d644300df2be450abdeb33426f0e65b4ae48",
```
Comment (jxom) [inline src/experimental/eip5792/actions/writeContracts.test.ts:201]: please revert this

File: src/chains/definitions/xrSepolia.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    name: 'tXR',
+    symbol: 'tXR',
+  },
+  rpcUrls: {
+    default: { http: ['https://xr-sepolia-testnet.rpc.caldera.xyz/http'] },
+  },
+  blockExplorers: {
+    default: {
+      name: 'Blockscout',
+      url: 'https://xr-sepolia-testnet.explorer.caldera.xyz/',
```
Comment (jxom) [inline src/chains/definitions/xrSepolia.ts:17]:
```suggestion
      url: 'https://xr-sepolia-testnet.explorer.caldera.xyz',
```

**Comment by RealWooblay** [comment]
@jxom Please review for merge

File: src/experimental/eip5792/actions/writeContracts.test.ts
Comment (jxom) [inline src/experimental/eip5792/actions/writeContracts.test.ts:1]: These changes still need to be reverted.

**Review by jxom — CHANGES_REQUESTED** [review]

**Review by jxom — APPROVED** [review]

**Pull request merged**